docs/intro/what.md

```diff
@@ -3,12 +3,28 @@ title: 🔮 What is SourceCred?
 description: Overview of what SourceCred is and how it works
 ---
 
-Everything people do to support a project—like writing code, filing bug reports, organizing meetups, or even mediating hard discussions—is considered a contribution.
+## A Social Algorithm
 
-These contributions are connected to each other based on their relationships. For example, a pull request may fix a particular bug, and a code review may review that pull request.
+SourceCred enables communities to track **cred**, a quantitative metric that shows how much value a contribution
+or contributor has added to the community. Cred is calculated by tracking the contributions of a project--and
+every contribution that has gone into it--into a network or **contribution graph**. Contributions in the graph earn cred
+if they are connected to other important contributions, and contributors earn cred if they create
+high-cred contributions.
 
-This network of contributions and relationships form a contribution graph. SourceCred applies the [PageRank algorithm](https://en.wikipedia.org/wiki/PageRank)  to this graph, generating a score, called cred for every contribution.
+## An Economic System
 
-The basic idea is: a contribution earns cred if it is connected to other contributions that earn lots of cred. Contributors are connected to the contributions they've helped with, which means that they also earn cred.
+Once a community is up-and-running with cred, they can begin to issue **grain**. Grain is a community-controlled
+digital currency that is distributed based on cred scores. Thus, a contributor earns grain, and a sponsor can
+support that contributor by buying their grain.
 
-The project's community and maintainers have a lot of influence over this process. They can reconfigure SourceCred weights and parameters. For example, they could assign extra weight to some very important contributions, or decrease the weights of contributions that seem spammy. They can also add heuristics to do this automatically.
+Grain represents your skin in the game within a community. You can use your grain to vote on values, and to set priorities.
```

Comment:
>You can use your grain to vote on values, and to set priorities.

This comma is making me twitch...Suggest rewording for flow, unless it breaks the meaning.

>You can use your grain to vote on values and set priorities.